eiman.php

```diff
@@ -2,167 +2,167 @@
-                <span class="navbar-toggler-icon"></span>
-            </button>
-            <div class="collapse navbar-collapse" id="navbarNavDropdown">
-                <ul class="navbar-nav">
-                    <li class="nav-item">
-                        <a class="nav-link active" aria-current="page" href="eiman.php">Home</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="hobbies.htm">Hobbies</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="madlibs.php">Madlibs</a>
-                    </li>
-                    <li class="nav-item dropdown">
-                        <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">References</a>
-                        <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
-                            <li><a class="dropdown-item" href="https://www.w3schools.com/" target="_blank">W3Schools</a></li>
-                            <li> <a class="dropdown-item" href="https://www.freecodecamp.org" target="_blank">FreeCodeCamp</a></li>
-                            <li> <a class="dropdown-item" href="https://www.codeacademy.com" target="_blank">Codeacademy</a></li>
-                        </ul>
-                    </li>
-                </ul>
-            </div>
-        </div>
-    </nav>
+	<nav class="navbar container-fluid sticky-top navbar-expand-lg navbar-dark bg-dark">
+		<div class="container-fluid">
+			<a class="navbar-brand" href="#">Welcome</a>
+			<button class="navbar-toggler btn btn-outline-secondary active" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+				<span class="navbar-toggler-icon"></span>
+			</button>
+			<div class="collapse navbar-collapse" id="navbarNavDropdown">
+				<ul class="navbar-nav">
+					<li class="nav-item">
+						<a class="nav-link active" aria-current="page" href="eiman.php">Home</a>
+					</li>
+					<li class="nav-item">
+						<a class="nav-link" href="hobbies.htm">Hobbies</a>
+					</li>
+					<li class="nav-item">
+						<a class="nav-link" href="madlibs.php">Madlibs</a>
+					</li>
+					<li class="nav-item dropdown">
+						<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">References</a>
+						<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
+							<li><a class="dropdown-item" href="https://www.w3schools.com/" target="_blank">W3Schools</a></li>
+							<li> <a class="dropdown-item" href="https://www.freecodecamp.org" target="_blank">FreeCodeCamp</a></li>
+							<li> <a class="dropdown-item" href="https://www.codeacademy.com" target="_blank">Codeacademy</a></li>
+						</ul>
+					</li>
```

Comment:
This whole section needs proper indenting

Reply:
This isn't showing up in VScode. Looks good in the file.